hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1020,6 +1020,30 @@ public OzoneSnapshot getSnapshotInfo(String volumeName,
     return OzoneSnapshot.fromSnapshotInfo(snapshotInfo);
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public OzoneSnapshot getSnapshotInfo(String volumeName,
+                                       String bucketName,
+                                       String snapshotName,
+                                       String omNodeId) throws IOException {
```

Comment:
Thanks for the patch @ivandika3. Can we have a single snapshot function definition and have clients set null omNodeIds if `--om-node-id` is not specified?

Reply:
Thank you for the review. Previously, I have two different APIs to be more explicit to the client that if `omNodeId` is specified, it will be definitely sent to that OM node and not the leader.

However, I think it's better to unify it.